PKHeX.WinForms/Util/WinFormsUtil.cs

```diff
@@ -228,8 +228,8 @@ public static bool OpenSAVPKMDialog(string[] Extensions, out string path)
             string pathCache = CyberGadgetUtil.GetCacheFolder();
             if (Directory.Exists(pathCache))
                 cgse = Path.Combine(pathCache);
-            if (!PathUtilWindows.DetectSaveFile(out path, cgse))
-                Error(path);
+            if (!PathUtilWindows.DetectSaveFile(out path, cgse) && !string.IsNullOrEmpty(path))
```

Comment:
Looking more closely, path will never be null or empty when the function returns false. It may be a good idea to keep this here in case the function changes, but I can revert this part if you want. The important parts of this PR are really the comments.